.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Create a report to help us improve
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ If you do not respect this template your issue will be closed.
```

Comment:
Please remove this. I do not think this warning is necessary in this repository so far. I would like to keep as much as possible a minimum level of flexibility and friendliness. In case of (very rare) problem, we can close/suppress the issue anyway.

Reply:
This warning was in the previous template and added by you in #333

Reply:
Then my 2021 self disagrees with my 2019 self, and right now I stand with my 2021 self (mainly for temporal consistency reasons). Seriously, I think it should be removed, and this PR is an opportunity to do that.